### ISSUE TITLE
Screen Readers: Workspaces: The table row is announced as 'row', without displayed details

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76913]
## Action Performed:
Prerequisites:
- the user is signed in

1. Using Chrome + JAWS, open the URL https://new.expensify.com/
2. Navigate using the Tab key to the Workspaces section.
3. Navigate using the down arrow key to the visually apparent table elements and observe the screen reader announcement.

## Expected Result:
Each row should have a clear, descriptive label summarizing its key information, e.g:
- 'Workspace: Andris Garkalns, Owner: John, Type: Control'

## Actual Result:
Screen reader announces: “row, button”. The row contents, such as Workspace name, Owner, and Workspace type, are not announced, preventing users from understanding which workspace they are interacting with.

HTML:
<button aria-label="row" role="button" data-selection-scrapper-hidden-element="true" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-bhtmuz r-9ffhgg r-lrvibr" type="button" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><div class="css-g5y9jx r-18u37iz r-hcv6sl r-1q9bdsx"><div class="css-g5y9jx r-13awgt0 r-18u37iz r-16pcm1t r-1pcd2l5" data-component="AnimatedComponent" data-element="Component" data-source-file="createAnimatedComponent.js" style="background-color: rgba(0, 0, 0, 0); height: auto; opacity: 1; border-radius: 8px;"><div class="css-g5y9jx r-18u37iz r-13awgt0 r-16pcm1t"><div class="css-g5y9jx r-18u37iz r-1wtj0ep r-dta0w2 r-1awozwy"><div class="css-g5y9jx r-18u37iz r-1ssbvtb r-13awgt0 r-1awozwy"><div class="css-g5y9jx r-633pao" data-testid="Avatar"><div class="css-g5y9jx r-1niwhzg r-1kihuf0" style="height: 40px; width: 40px; border-radius: 40px;"><div class="css-g5y9jx" data-testid="SvgDefaultAvatar_a Icon" style="overflow: hidden; border-radius: 8px; background-color: rgb(1, 133, 255); fill: rgb(0, 60, 115);"><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 80 80" width="40" height="40" hovered="false" pressed="false" fill="#003C73"><path d="..."></path></svg></div></div></div><div dir="auto" class="css-146c3p1 r-dnmrzs r-1udh08x r-1udbk01 r-3s2u2q r-1iln25a" style="color: rgb(231, 236, 233); font-size: 15px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 700; flex: 1 1 0%; line-height: 20px;">andris.garkalns</div></div></div><div class="css-g5y9jx r-18u37iz r-13awgt0 r-11v9fek r-1cmwbt1 r-1awozwy"><div class="css-g5y9jx r-5scogr r-633pao" data-testid="Avatar"><div class="css-g5y9jx"><div class="css-g5y9jx" data-testid="SvgFallbackAvatar Icon" style="overflow: hidden; border-radius: 100px; background-color: rgb(34, 79, 65); fill: rgb(139, 156, 143);"><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 40 40" width="28" height="28" hovered="false" pressed="false" fill="#8B9C8F"><path d="..."></path></svg></div>

## Area issue was found in:
Workspace

## Failed WCAG checkpoints
WCAG 2.4.6 Headings and Labels
https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels

## User impact:
Descriptive headings are necessary for users who have disabilities that make reading slow and for people with limited short-term memory. These users benefit when section titles make it possible to predict what each section contains. Form input controls with labels that clearly describe the content that is expected to be entered helps users know how to successfully complete the form.

## Suggested resolution:
Replace generic aria-label="row" with a meaningful, dynamic label that summarizes the row data, e.g.:
- 'Workspace: Andris Garkalns, Owner: John, Type: Control '
https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA14

## Environment(s)
Windows Windows 11 - Chrome

## Screenshots/Videos
https://github.com/user-attachments/assets/e088227e-0d6b-45ee-9f3b-9a2dbb2b199b

<img width="1919" height="926" alt="Image" src="https://github.com/user-attachments/assets/99b22868-bb9e-4958-a1ec-603a1b5b2a4c" />

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6999287
**Issue reported by:** NA
**Slack conversation:** NA
